pilot/pkg/config/kube/ingress/controller_test.go

```diff
@@ -183,11 +184,10 @@ func TestIngressController(t *testing.T) {
 
 	// Append an ingress notification handler that just counts number of notifications
 	stop := make(chan struct{})
-	count := make(chan bool)
-	defer close(count)
 
+	numNotifications := atomic.NewInt64(0)
```

Comment:
please don't use atomic. see the reasons below.

Reply:
This seems like a totally legitimate use of atomic ops. Channels would end up considerably more complex, obfuscated, and higher overhead.